python/example_robot_data/robots_loader.py

```diff
@@ -161,11 +175,27 @@ def loadANYmal(withArm=None):
 
 class CassieLoader(RobotLoader):
     path = 'cassie_description'
-    sdf_filename = "cassie_v2.sdf"
+    if tuple(int(i) for i in pin.__version__.split('.')) > (2, 9, 1):
+        sdf_filename = "cassie.sdf"
+    else:
+        sdf_filename = "cassie_v2.sdf"
```

Comment:
```suggestion
    sdf_filename = "cassie.sdf" if tuple(int(i) for i in pin.__version__.split('.')) > (2, 9, 1) else "cassie_v2.sdf"
```

Reply:
(I don't think you can actually execute code in the class declaration, but you can execute code in the parameter declaration… But I might be wrong)

Reply:
my bad, this works, I wasn't aware of it